core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2253,6 +2253,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_READ_CACHE_TTL =
+      new Builder(Name.USER_FILE_READ_CACHE_TTL)
```

Comment:
I think this applies only to newly loaded files, not just and read files

Reply:
Is USER_FILE_LOAD_TTL good, remove "CACHE" as well?

Reply:
If we use LOAD, is it ok to rename "WRITE" to "CREATE". I think it should try to keep symmetrical, like WRITE <-> READ, CREATE <-> LOAD.